src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-user-list/rgw-user-list.component.ts

```diff
@@ -149,6 +149,7 @@ export class RgwUserListComponent extends ListWithDetails implements OnInit {
 
   deleteAction() {
     this.modalService.show(CriticalConfirmationModalComponent, {
+      impact: DeletionImpact.high,
```

Comment:
Should users be high impact?

@cloudbehl thoughts?

Reply:
yes, deleting rgw users should be as thats like deleting the user id.